nixos/tests/web-apps/peertube.nix

```diff
@@ -24,6 +24,7 @@ import ../make-test-python.nix ({pkgs, ...}:
           CREATE ROLE peertube_test LOGIN PASSWORD '0gUN0C1mgST6czvjZ8T9';
           CREATE DATABASE peertube_local TEMPLATE template0 ENCODING UTF8;
           GRANT ALL PRIVILEGES ON DATABASE peertube_local TO peertube_test;
+          ALTER DATABASE peertube_local OWNER TO peertube_test;
```

Comment:
We should use ensureDBOwner

Reply:
The `ensureDBOwnership` instruction is converted to a similar sql command.

Reply:
but that's what people are likely using, so the test would be accurate.

Also the database should be default utf8.

Reply:
Ok, fixed.